### DISCUSSION TITLE
Add missing solution for LC 437 (https://leetcode.com/problems/path-sum-iii/)

### DISCUSSION THREAD

**Discussion by kesava-karri** [wingkwong/leetcode-the-hard-way#682] [Q&A] [answered]
Hello there,
Are others allowed to submit solutions in this repo for LC problems?

* My approach was extending the path sum ii logic and using it along with a queue (BFS style).
* Please let me know if we can do this.

TIA.

Problem link: [here](https://leetcode.com/problems/path-sum-iii/)

**Comment by wingkwong** [comment, marked as answer]
Yes. Please take a look at https://github.com/wingkwong/leetcode-the-hard-way/blob/main/CONTRIBUTING.md.

**Reply by kesava-karri** [reply]
Thank you for sharing this detailed information. I'll take a look at it and get back to you if I have any questions.

**Comment by vkaushik2607** [comment]
is it possible to post Linked List tutorial and tricks and solutions in C++ under "Basic Topics" category ?

**Reply by wingkwong** [reply]
@vkaushik2607 

1. Linked List tutorial is already [there](https://leetcodethehardway.com/tutorials/basic-topics/linked-list). You can add more content to this page.
2. For solutions, it should be under [solution section](https://leetcodethehardway.com/solutions/category/0000---0099).

Free feel to DM me at Discord for discussion.
